UnleashClient/strategies/FlexibleRolloutStrategy.py

```diff
@@ -29,6 +29,7 @@ def apply(self, context: dict = None) -> bool:
         elif stickiness == "random":
             calculated_percentage = self.random_hash()
         else:
-            calculated_percentage = normalized_hash(context[stickiness], activation_group)
+            custom_stickiness = context.get(stickiness) or context.get("properties")[stickiness]
```

Comment:
Fixes an issue with stickiness where you have nested properties. I've allowed the exception raised by the null access here to propagate here when stickiness is not found on the dict rather than allow it to crash later